AppCenter/AppCenterSDK+Mac.swift

```diff
@@ -0,0 +1,164 @@
+//
```

Comment:
Do we need that file ? What are the differences with `AppCenterSDK+HiDrive` ?
I would assume that we use the same implementation, better reduce the number of files to maintain.

Reply:
The problem is, that the AppCenterDistribute module is not available for mac .. and as you don't need any distribution logic i just removed the import statement and any associated code

Reply:
What about extracting the Distribution code in an extension in the same file, and surrounding it by `#if !os(macOS)` ?

(Not sure if it actually works, just a random idea)

Reply:
doesn'T sound that bad i can try that

Reply:
done!